setup.py

```diff
@@ -56,21 +57,15 @@ def dependency_links(filename):
             for line in open(join(ROOT, 'requirements', filename))
             if '://' in line]
 
-
-long_description = '\n'.join((
-    rst('README.rst'),
-    rst('CHANGELOG.rst'),
-    ''
-))
-
 install_requires = pip('install.pip')
 tests_require = pip('test.pip')
+readme_md = clean_doc('README.md')
 
 setup(
     name='udata',
     version=__import__('udata').__version__,
     description=__import__('udata').__description__,
-    long_description=long_description,
+    long_description=pypandoc.convert(readme_md, 'rst', format='md'),
```

Comment:
Or third solution: embed the long_description directly into the setup.py. It won't be extracted anymore from the README, but who cares, we don't publish on PyPI right now :)